src/app/markets/page.tsx

```diff
@@ -1,3 +1,37 @@
+"use client";
+
+import { useEffect } from "react";
+
+import { DisplayMarketTable } from "components/markets/MarketsTable";
+import { MarketsInfo } from "components/markets/MarketsInfo";
+
+import { initializeSubscriptions, unsubscribeAll } from "../subscriptions";
+
+import { store } from "../state/store";
+
 export default function Markets() {
-  return <div className="col-span-full p-4">Markets</div>;
+  useEffect(() => {
+    initializeSubscriptions(store);
+    return () => {
+      unsubscribeAll();
+    };
+  }, []);
+
+  return (
+    <div className="col-span-full p-4 mx-10">
+      <div className="flex mx-5 p-8 justify-between items-center uppercase border-b-2 border-b-secondary-content">
+        <div className="flex-none">
+          <MarketsInfo />
+        </div>
+        <div className="flex-last">
+          <button className="btn btn-lg uppercase bg-neutral text-accent border-x-4 border-y-0 border-accent">
+            Trade Now
+          </button>
```

Comment:
This button does not do anything. Maybe removing it to not create confusion? We already have the TRADE button in the navbar.